### ISSUE TITLE
Cannot import name '_legacy_validators' from 'jsonschema' 

### ISSUE THREAD

**Issue by mielvds** [OnroerendErfgoed/atramhasis#889]
Running a custom scaffold of atramhasis gives me the following error at startup:

```
2024-07-08 16:28:09   File "/app/atramhasis/meemoo_atramhasis/__init__.py", line 23, in main
2024-07-08 16:28:09     config.include("atramhasis")
2024-07-08 16:28:09   File "/usr/local/lib/python3.11/site-packages/pyramid/config/__init__.py", line 666, in include
2024-07-08 16:28:09     c(configurator)
2024-07-08 16:28:09   File "/usr/local/lib/python3.11/site-packages/atramhasis/__init__.py", line 18, in includeme
2024-07-08 16:28:09     config.include("pyramid_openapi3")
...
2024-07-08 16:28:09   File "/usr/local/lib/python3.11/site-packages/openapi_schema_validator/shortcuts.py", line 3, in <module>
2024-07-08 16:28:09     from openapi_schema_validator.validators import OAS30Validator
2024-07-08 16:28:09   File "/usr/local/lib/python3.11/site-packages/openapi_schema_validator/validators.py", line 3, in <module>
2024-07-08 16:28:09     from jsonschema import _legacy_validators, _utils, _validators
2024-07-08 16:28:09 ImportError: cannot import name '_legacy_validators' from 'jsonschema' (/usr/local/lib/python3.11/site-packages/jsonschema/__init__.py)
```

This is probably caused by a renaming in `jsonschema > 4.19.0` (see https://github.com/python-openapi/openapi-schema-validator/issues/131). Some mismatch between Pyramid, [pyramid_openapi3](https://github.com/Pylons/pyramid_openapi3) and jsonschema?

This issues was supposily fixed in openapi-schema-validator 0.6.2, but I also noticed the following during the build of atramhasis:

```
INFO: pip is looking at multiple versions of openapi-core to determine which version is compatible with other requirements. This could take a while.
```

and it ends up with openapi-schema-validator-0.2.3

**Comment by goessebr** [comment]
Hi @mielvds,

Could you check if you have version 2.1.0a2 installed in your venv?

First, activate your virtual environment if it's not already active. To check if you have version 2.1.0a2 of atramhasis installed, run:
`pip freeze | grep atramhasis`

If you have a lower version installed, then following command will fix the error for you:
`pip install atramhasis==2.1.0a2`

When version 2.1.0 is officially released, pip will automatically install the correct version during a `pip install -e .`. But right now pip will opt for the latest official release which is 2.0.0 unless you manually install a newer version.

**Comment by mielvds** [comment]
Thanks @goessebr . The issue was indeed resolved with `pip install atramhasis==2.1.0a2`. Oddly enough `pip freeze` did not return any entry for atramhasis. Maybe add this somewhere in https://atramhasis.readthedocs.io/en/latest/customisation.html#creating-your-own-project?

**Comment by goessebr** [comment]
Hi @mielvds,

As soon as you run pip install -e . in your virtual environment, Atramhasis should always be present since it is a required package. I'm not sure what happened there. The error you encountered is from an old version of Atramhasis, so it must have been there somewhere.
